wasm/jit/jit_arm64.go

```diff
@@ -862,19 +862,113 @@ func (c *arm64Compiler) compileXor(o *wazeroir.OperationXor) error {
 }
 
 func (c *arm64Compiler) compileShl(o *wazeroir.OperationShl) error {
```

Comment:
just a quick comment and will review tomorrow. when we implement functions we have been adding godoc to each so that tech debt doesn't pile up. can you look at some that @mathetake recently committed and add docs similarly? thanks tons!

Reply:
ps @mathetake don't block on me if you end up reviewing before I'm around. merge when you feel ready

Reply:
Thanks for the feedback! 
Added minimum document in 856d7df.
If more detailed comments need to be added, I would be happy to point them out.